package.json

```diff
@@ -31,7 +31,7 @@
     "devserver:hot": "npm-run-all --parallel build:dev:hot runserver",
     "devserver-hot": "yarn run devserver:hot",
     "devshell": "cd contentcuration && python manage.py shell --settings=contentcuration.dev_settings",
-    "celery": "(cd contentcuration && DJANGO_SETTINGS_MODULE=contentcuration.dev_settings celery -A contentcuration worker --without-mingle --without-gossip -l info) || true",
+    "celery": "(cd contentcuration && DJANGO_SETTINGS_MODULE=contentcuration.dev_settings celery -A contentcuration worker --without-mingle --without-gossip -c 1 -l info) || true",
```

Comment:
In production we are using `--concurrency=3`, also in `devceleryworkers` we use the same setting. So maybe we should use `--concurrency=3` here as well?

https://github.com/learningequality/studio/blob/b9729c5bc9d9ae4a43855dc110c936db8ead6f0d/Makefile#L19

Reply:
Okay sounds good!